src/main/java/bisq/asset/coins/Zero.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.AddressValidationResult;
+import bisq.asset.AddressValidator;
+import bisq.asset.Coin;
+
+public class Zero extends Coin {
+
+    public Zero() {
+        super("Zero", "ZER", new ZeroAddressValidator());
+    }
+
+
+    public static class ZeroAddressValidator implements AddressValidator {
+
+        @Override
+        public AddressValidationResult validate(String address) {
+            // We only support t addresses (transparent transactions)
+            if (!address.startsWith("t1"))
```

Comment:
Is that the only validation requirement?

Is "t" address valid?

Reply:
yes that is the only "Validation requirement".

Reply:
For us, same as Zcash, only transactions to T addresses can be readily verified. and valid addresses start with "t1".